test/e2e/daemon_set.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 const (
-	updateRetryPeriod    = 5 * time.Second
+	updateRetryPeriod    = 2 * time.Second
```

Comment:
nit: Please add a comment to explain why this should not be the same as the node status update interval, as per your description of this PR, so that it doesn't get lost in the mists of time.